crates/biome_css_formatter/src/css/selectors/complex_selector.rs

```diff
@@ -23,16 +23,47 @@ impl FormatNodeRule<CssComplexSelector> for FormatCssComplexSelector {
                 write!(f, [combinator.format()])
             }
         });
+        let mut has_leading_comments = false;
 
-        write!(
-            f,
-            [
-                left.format(),
-                soft_line_break_or_space(),
-                formatted_combinator,
-                space(),
-                right.format()
-            ]
-        )
+        let is_selector_list_first_child = node.syntax().parent().is_some_and(|parent| {
+            matches!(parent.kind(), CssSyntaxKind::CSS_SELECTOR_LIST)
+                && node.syntax().prev_sibling().is_none()
+        });
```

Comment:
Can you please add a comment explaining why we have this chack? Also, can you use a better name?

Reply:
ok, i will add this later~